runtime/polkadot/src/lib.rs

```diff
@@ -1630,7 +1629,7 @@ impl FixCouncilDepositMigration {
 
 impl OnRuntimeUpgrade for FixCouncilDepositMigration {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		if VERSION.spec_version == 9150 {
+		if VERSION.spec_version == 9180 {
```

Comment:
I think I already said this once, but do we really want this? I mean this now already failed once because of this :P

Reply:
better not execute something that execute it twice. 

This was originally added in 9150, and no one removed it (the blame for which is on me as well) until now. If we didn't have had this `if`, we would have executed this 3 times already in 9.15, 9.16, and 9.17 :D